### ISSUE TITLE
Problem in Cms\Classes\Theme::getDatasource()

### ISSUE THREAD

**Issue by webmaxx** [wintercms/winter#855]
### Winter CMS Build

dev-develop

### PHP Version

8.1

### Database engine

MySQL/MariaDB

### Plugins installed

Wintercms.Pages, Wintercms.Sitemap, Wintercms.Notify, Wintercms.Redirect

### Issue description

```php
/**
 * Get the theme's datasource
 */
public function getDatasource(): AutoDatasource
{
    $resolver = App::make('halcyon');
    return $resolver->datasource($this->getDirName());
}
```
Cms\Classes\Theme::getDatasource(): Return value must be of type Cms\Classes\AutoDatasource, Winter\Storm\Halcyon\Datasource\FileDatasource returned

### Steps to replicate

1. Create new theme in console `artisan create:theme example`
2. Activate new theme in backend (and DO NOT change anything in "Customize theme" or click on the button "Reset to default")
3. Go to main page

**Comment by webmaxx** [comment]
Made a video demonstration: https://youtu.be/EHFv70Csqs0

**Comment by jaxwilko** [comment]
Hi @webmaxx could you do me a favour and try https://github.com/wintercms/winter/pull/856 to see if that fixes the issue for you? I couldn't actually get it to fail on my end but that's probably a configuration issue for me

**Comment by webmaxx** [comment]
> Hi @webmaxx could you do me a favour and try #856 to see if that fixes the issue for you? I couldn't actually get it to fail on my end but that's probably a configuration issue for me

Yes. It's working

**Comment by LukeTowers** [comment]
Fixed by #856
